lib/src/files.rs

```diff
@@ -12,14 +12,15 @@
 // See the License for the specific language governing permissions and
```

Comment:
Looks like the commit message is out of date. We don't remove these tests any more.

Reply:
I think I've updated it. You're referring to https://github.com/martinvonz/jj/pull/1610/commits/d10895df55d6d20ea0c3cb1b0b1382d47ee0cfd6, right?

Reply:
Bleh GitHub. I was referring to fdddbb8160dd5b87d12fb3b58daa26582434ae05.

Reply:
Fixed.